qutebrowser/config/configdata.yml

```diff
@@ -1325,7 +1325,10 @@ tabs.show:
 
 tabs.show_switching_delay:
   default: 800
-  type: Int
+  type:
+    name: Int
+    minval: -2147483648
+    maxval: 2147483647
```

Comment:
I think it would be a bit cleaner to set this to `maxint` rather than the literal value of the maximum int. [see this for an example](https://github.com/AlternateData/qutebrowser/blob/a52d18b700df262945384b70f1ffb58fc2fa0360/qutebrowser/config/configdata.yml#L195).